calcom/cal.com
# fix: phone number disappearing issue

Fixes the pre-fill issue for Phone number field while booking event or rescheduling event

Fixes https://github.com/calcom/cal.com/issues/9682

## Before Fix
https://www.loom.com/share/a6024364e1c744bf9db82e4b62504514?sid=8bb91e81-8055-474a-948e-ae7bc07520ff

## After Fix
https://www.loom.com/share/6ee121cf728d4769a915d7ce2172795e?sid=72a86bcb-84cf-41d3-9ef5-e7303a21358a

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

-  Create an event type with options Phone number field (By going to advanced settings)
-  Choose `Identifier` as `phone` or whatever you want to fill
-  Open the URL for that event in another browser or in incognito mode by pre-filling the phone number identifier in the URL
-  You will see now phone number will be pre-filled for you and it will not disappear.
-  After you have created schedule, then try to reschedule the same event. It should pre-fill the phone number in that case too 

## Mandatory Tasks
